app/openai/mcp_client.py

```
@@ -0,0 +1,145 @@
+        async with stdio_client(self.server_params) as (read, write):
+            async with ClientSession(read, write) as session:
+                await session.initialize()
+                yield session
+
+    async def get_pet_by_id(self, pet_id: int):
+        """ID로 펫 정보 조회"""
+        async with self.create_session() as session:
+            try:
+                result = await session.call_tool(
+                    "execute_sql",
+                    {"query": f"SELECT * FROM pet WHERE id = {pet_id}"}
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)